lib/srv/ctx.go

```diff
@@ -379,6 +379,9 @@ type ServerContext struct {
 
 	// JoinOnly is set if the connection was created using a join-only principal and may only be used to join other sessions.
 	JoinOnly bool
+
+	// NodeSubKind if the sub kind of the node this context is for.
+	NodeSubKind string
```

Comment:
nit: I would stick to the "server" naming, since this is called `ServerContext`.
```suggestion
	// ServerSubKind if the sub kind of the node this context is for.
	ServerSubKind string
```

Reply:
Good call, done